packages/workbox-build/src/index.js

```diff
@@ -7,9 +7,16 @@ const injectManifest = require('./lib/inject-manifest');
 /**
  * # workbox-build
  *
- * This Node module can be used to generate a list of assets that should be
- * precached in a service worker, generating a hash that can be used to
- * intelligently update a cache when the service worker is updated.
+ * A module that integrates into your build process, helping you generate a
+ * manifest of local files that [`workbox-sw`](../workbox-sw) should precache.
```

Comment:
This link worries me a little. Where should it be pointing to? If it's JSDoc then please use the expected syntax - of the top of my head, untested, I think: `[workbox-sw]{@link module:workbox-sw}` should work.